package.json

```diff
@@ -2,13 +2,18 @@
   "name": "bloom-housing",
   "version": "0.0.10",
   "description": "Bloom is a system to manage processes for affordable housing",
-  "workspaces": [
-    "shared/core",
-    "shared/ui-components",
-    "apps/public-reference",
-    "services/listings"
-  ],
-  "repository": "https://github.com/Exygy/bloom.git",
+  "workspaces": {
+    "packages": [
+      "shared/core",
+      "shared/ui-components",
+      "apps/public-reference",
+      "services/listings"
+    ],
+    "nohoist": [
+      "**/@types/mocha", "**/@types/jest", "**/@types/gl-matrix"
+    ]
+  },
+ "repository": "https://github.com/Exygy/bloom.git",
```

Comment:
Shouldn't that be https://github.com/bloom-housing/bloom ?

Reply:
I did not change it. It's marked as change because there is a slight indent misalignment